### PR TITLE
Add docs for Turn Touch component.

### PR DESCRIPTION
**Pull request by antsar**
**Description:** Add documentation for the [Turn Touch](https://shop.turntouch.com/) component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** home-assistant/home-assistant#15126

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/turn_touch.markdown [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+```
+
+{% configuration %}
+devices:
+  required: true
+  description: A list of Turn Touch devices.
+  type: list
+{% endconfiguration %}
+
+Configuration variables for each device:
```
Comment (frenck) [inline source/_components/turn_touch.markdown]: Please include these inside the configuration block you've defined above.

File: source/_components/turn_touch.markdown [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+- **mac** (*Required*): The MAC address of your Turn Touch remote.
+- **debounce** (*Optional*): Enable debouncing. Default: `true`.
+
+## {% linkable_title Debouncing %}
+
+Turn Touch supports complex button presses (double-tap, hold, and multi-button). Complex button presses generate multiple events (eg. a double-tap generates two single-press events, followed by the double-tap event). Normally, the single-press events are suppressed by the `debounce` setting. However, this also introduces a 0.75-second delay to all button presses.
```
Comment (frenck) [inline source/_components/turn_touch.markdown]: `eg.` -> `e.g.,`

File: source/_components/turn_touch.markdown [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+The `turn_touch` component offers integration with the [Turn Touch](https://shop.turntouch.com/) smart remote. It provides events for button presses as well as a [sensor](/components/sensor.turn_touch) for the remote's battery level.
+
+To use Turn Touch, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+turn_touch:
+  devices:
+    - mac: '00:de:ad:be:ef:00'
+      debounce: false
```
Comment (frenck) [inline source/_components/turn_touch.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — CHANGES_REQUESTED** [review]
Hi, @antsar,

I've reviewed the PR and they look pretty good, nevertheless, I've left some comments. Could you please take a look? 👍

**Comment by antsar** [comment]
Thanks for the quick review, @frenck! Changes made.

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
No! Thank you for all this work! Awesome.
✅ Approved, can be merged as soon as the parent PR get merged.

bitmoji

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Closing this PR since the parent PR has been closed as well.

**Pull request closed without merging**